bitmark-references/bitmark-sdk/websocket.md

```diff
@@ -0,0 +1,85 @@
+# Web Socket
+Realtime event triggering is trending in modern application development nowadays. It helps the application is always up to date and so provides better user experience.
+Bitmark SDK offers Web Socket event triggering to help the application can be notified immediately when there are anything changes.
+
+## Event
+- New block: This event will be sent when bitmark blockchain has a new block.
+  + `blockNumber` : new block number
+- Bitmark changed: This event will be sent when users issue new bitmarks, received new bitmarks, or send them to new owner.
+  + `bitmarkId`: the id of bitmark is changed
+  + `txId`: the corresponding transaction id
+  + `presence`: indicate that the bitmark is owned or not by the corresponding `Account`
+- New transfer offer: This event will be sent when users receive new transfer offer (for 2 signatures transfer) from someone.
+  + `bitmarkId`: the id of bitmark is comming
+- New pending issuance: This event will be sent when users issue new bitmarks.
+  + `bitmarkId` : the id of bitmark is issuing
+- New pending transaction: This event will be sent to both sender and receiver.
```

Comment:
```suggestion
- New pending transfer?: This event will be sent to both sender and receiver.
```

Reply:
I think transaction is more reasonable. What do you think?

Reply:
Issue and share-relevant txs are also txs.

Reply:
It's fine